webview/cocoa.py

```diff
@@ -57,6 +57,16 @@ class BrowserDelegate(AppKit.NSObject):
         def webView_contextMenuItemsForElement_defaultMenuItems_(self, webview, element, defaultMenuItems):
             return nil
 
+        # Display a JavaScript alert panel containing the specified message
+        def webView_runJavaScriptAlertPanelWithMessage_initiatedByFrame_\
+                (self, webview, message, frame):
```

Comment:
Well the next line is very long anyway, so lets keep the style consistent. I tend to break a function definition line at parameters, breaking just after the name looks weird in my opinion.